src/App.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import './App.css';
-
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './Components/todolist/Todolist';
+
+import { preparedTodosList } from './Components/preparedTodosList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+const preparedList = preparedTodosList(todos, users);
```

Comment:
функцию нужно называть глаголом, действие

getWhat
setWhat

```suggestion
const preparedList = getPreparedTodos(todos, users);
```
```suggestion
const preparedList = getTodosWithUsers(todos, users);
```